site/public/css/global.css

```diff
@@ -189,3 +189,9 @@ footer .footer-separator {
 #loading-bar-percentage {
     margin-left: 10px;
 }
+
+#no-style-invisible-button {
```

Comment:
ID's should only be used once in a HTML file. They represent a unique item. I think this should be a class. See <https://css-tricks.com/the-difference-between-id-and-class/> for more details.